packages/library-base/src/commonMain/kotlin/io/realm/RealmList.kt

```diff
@@ -36,8 +37,18 @@ import kotlinx.coroutines.flow.Flow
  * that inject values into a class. Unmanaged elements in a list can be added to a Realm using the
  * [MutableRealm.copyToRealm] method.
  */
-interface RealmList<E> : MutableList<E> {
-    fun observe(): Flow<RealmList<E>>
+interface RealmList<E> : MutableList<E>, Flowable<RealmList<E>> {
+
+    /**
+     * Observes changes to the RealmList. If there is any change to the list, the flow will emit the
+     * updated RealmResult. The flow will continue running indefinitely until canceled.
```

Comment:
Is this right? Don't we cancel it if the parent object is deleted?

Reply:
That is correct, I will update it.